plugins/tiddlywiki/multiwikiclient/info-segment.tid

```diff
@@ -0,0 +1,8 @@
+title: $:/plugins/tiddlywiki/multiwikiclient/info-segment
+tags: $:/tags/TiddlerInfoSegment
+
+<$reveal type="nomatch" state=<<folded-state>> text="hide" tag="div" retain="yes" animate="yes">
+<div class="tc-subtitle">
```

Comment:
I think since reveal already creates an animated and retained DOM element, it's not needed to include DIV just to add a class. It should be OK to add the class to the reveal widget.